scripts/deployment/main.py

```diff
@@ -214,7 +215,11 @@ def main(
         chain_id = web3_l2.eth.chain_id
         assert chain_id == l2_config["chain_id"]
 
-        supported_target_chains = {l2["chain_id"]: l2["finalization_time"] for l2 in config["L2"]}
+        supported_target_chains = {
+            l2["chain_id"]: l2["finalization_time"]
+            for l2 in config["L2"]
+            if l2["chain_id"] != chain_id or chain_id == GANACHE_CHAIN_ID
+        }
```

Comment:
How has it been working so far, without explicitly mentioning ganache?

Reply:
I've been using
```
--config-file scripts/deployment/ganache-local.json --test-messenger
```
as options to the deployment script for some time now and it worked for me.

Reply:
You removed the if clause in your PR: https://github.com/beamer-bridge/beamer/pull/488/files#diff-c6d950f3bb92a40d8b76462f0c0f34448141fee95c49373f1e35d96960a48828R217

So I bring it back and make sure that it still works on local chains.

Reply:
Ah, I see, you needed to bring it in due to it being the actual topic of the PR -- and then you need to allow for ganache again. Understood. :+1: